tests/shaders.rs

```diff
@@ -265,7 +265,7 @@ fn get_program_binary() {
         Ok(bin) => bin
     };
 
-    assert!(binary.content.len() >= 1);
+    assert!(!binary.content.is_empty());
```

Comment:
I think the `is_empty` lint of clippy is misguided/harmful. The older code was shorter as well as clearer.